py/desitarget/myRF.py

```diff
@@ -90,7 +90,7 @@ def predict_proba(self):
 
     def loadForest(self, forestFileName):
         # loads forest
-        t = np.load(forestFileName, encoding='bytes')
+        t = np.load(forestFileName, encoding='bytes', allow_pickle=True)

```

Comment:
Given that `allow_pickle` has not been needed to load the RF files in the past, is it really needed now? In general, `.npz` files, although a little module/language specific, should be supported by `numpy.load()` for the foreseeable future. But, the pickled versions are less secure, less likely to fail with meaningful errors on erroneous data, and have an extra dependency (on the pickle module). I'd prefer it if the DR8 RF files didn't require `allow_pickle`.

Reply:
It seems that `numpy.save_compressed()` use pickle automatically with file heavier than 12 MB. This why, I needed that. According to the documentation, this just allows file decompression using pickle and does not change the rest.

Whatever, with new RF files which are lighter, it's not require. I **remove** it.